models/user/user.go

```diff
@@ -1191,6 +1191,11 @@ func GetAdminUser(ctx context.Context) (*User, error) {
 	return &admin, nil
 }
 
+// GetAdminUserCount returns the count of all administrator
+func GetAdminUserCount(ctx context.Context) (int64, error) {
```

Comment:
Maybe `CountAdminUsers` or reuse `CountUsers` with additional conditions in `CountUserFilter`.

Reply:
`CountUsers` would not return errors. Will it cause issues here?

ps: I added print log in it, so when there's an errors occurred, we can trace it.